Makefile

```diff
@@ -14,26 +14,25 @@ help:
 	@echo "  make install-deps   - Install required Python packages from misc/requirements.in"
 	@echo "  make build-ui       - Compile UI files, resources, and translations (with --force)"
 	@echo "  make build-wheel    - Build Python wheel package using setup.py"
-	@echo "  make install-wheel  - Install the generated wheel package using python -m installer"
+	@echo "  make install-wheel  - Install the generated wheel package using pip install"
 	@echo "  make clean          - Remove virtual environment, build artifacts, and marker files"
 
 $(BUILD_DIR):
 	mkdir -p $@
 
 $(VENV_DIR): $(BUILD_DIR)
-	python -m venv $(VENV_DIR)
+	python3 -m venv $(VENV_DIR)
 venv: $(VENV_DIR)
 
 $(BUILD_DIR)/.rare-install-deps: venv
 	. $(VENV_DIR)/bin/activate && pip install -r misc/requirements.in
-	. $(VENV_DIR)/bin/activate && pip install ruff
```

Comment:
I re-added this on purpose because I removed it from `requirements.in`. This is still a stop-gap solution until I find the time and willingness to fully transition the scripts into setuptools build functionality.

Reply:
Thank you